service/src/main/java/eu/europa/ec/fisheries/uvms/user/service/dao/OrganisationDaoImpl.java

```diff
@@ -0,0 +1,39 @@
+package eu.europa.ec.fisheries.uvms.user.service.dao;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.persistence.PersistenceContext;
+
+import eu.europa.ec.mare.usm.information.entity.OrganisationEntity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.persistence.EntityManager;
+import javax.persistence.criteria.*;
+import java.util.List;
+
+@ApplicationScoped
+public class OrganisationDaoImpl implements OrganisationDao{
+
+    private static final Logger logger = LoggerFactory.getLogger(OrganisationDaoImpl.class);
+
+    @PersistenceContext(unitName = "USM-Administration")
```

Comment:
Let's create an `EntityManagerProducer` - Iakovos has implemented this I think, so maybe you can base your work on his commits.